### PR TITLE
Fixed Some Bugs

### PR DESCRIPTION
**Pull request by Kinplemelon**

### CODE REVIEW

File: src/components/TextareaWithUploader.vue [hunk 1]
```diff
@@ -119,8 +119,8 @@ const handleError = (error: any) => {
   .el-button {
     font-weight: normal;
     font-size: 14px;
-    background-color: #f7f7f7;
-    border-color: #f7f7f7;
+    background-color: --color-bg-primary;
+    border-color: --color-border-primary;
```
Comment (ysfscream) [inline src/components/TextareaWithUploader.vue:123]: Don't we need to use the `var` function here??

```css
background-color: var(--color-bg-primary);
```

Comment (Kinplemelon) [reply src/components/TextareaWithUploader.vue:123]: oh yes!

Comment (Kinplemelon) [reply src/components/TextareaWithUploader.vue]: fixed it, PTAL

Comment (ysfscream) [reply src/components/TextareaWithUploader.vue]: cool

**Review by ysfscream — APPROVED** [review]

**Pull request merged**